src/components/Landing/Payments.tsx

```diff
@@ -1,7 +1,7 @@
 import RichNote from 'components/shared/RichNote'
 import FormattedAddress from 'components/shared/FormattedAddress'
 import Loading from 'components/shared/Loading'
-import ProjectHandle from 'components/shared/ProjectHandle'
+import V1ProjectHandle from 'components/v1/shared/V1ProjectHandle'
```

Comment:
For clarity reasons. We'll need to come up with a strat for V2 (probably ENS name). Future problem.